### PR TITLE
Add rockbox utility v1.5.1

### PR DESCRIPTION
**Pull request by itfarrier**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/rockbox-utility.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  url "https://download.rockbox.org/rbutil/macosx/RockboxUtility-v#{version}.dmg"
+  name "Rockbox Utility"
+  desc "Automated installer tool for Rockbox"
+  homepage "https://www.rockbox.org/wiki/RockboxUtility"
+
+  livecheck do
+    url :homepage
+    strategy :page_match
+    regex(/RockboxUtility-v(\d+(?:\.\d+)*)\.dmg/i)
```
Comment (neutric) [inline Casks/rockbox-utility.rb:13]:
```suggestion
    regex(/RockboxUtility[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```

File: Casks/rockbox-utility.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  app "RockboxUtility.app"
```
Comment (neutric) [inline Casks/rockbox-utility.rb:16]: Please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

File: Casks/rockbox-utility.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+cask "rockbox-utility" do
+  version "1.5.1"
+  sha256 "b979b6aa7ab2894f32425114213d7f9efa803ee41523df6658739b3029058841"
+
+  url "https://download.rockbox.org/rbutil/macosx/RockboxUtility-v#{version}.dmg"
+  name "Rockbox Utility"
+  desc "Automated installer tool for Rockbox"
```
Comment (neutric) [inline Casks/rockbox-utility.rb:7]:
```suggestion
  desc "Automated installer for the Rockbox digital music player firmware"
```

**Review by neutric — COMMENTED** [review]
Thanks for your contribution to `homebrew-cask`, @itfarrier! My review is in the code. Take it with a grain of salt as I am not a maintainer.

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @itfarrier.

**Pull request merged**